doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,50 @@
 
-Depending on your needs, you can deploy either a single authoring environment or a high-availability (HA) authoring environment.
+You can use {CENTRAL} both to develop services and to deploy them to {KIE_SERVERS}. You can connect several {KIE_SERVERS} to one {CENTRAL} to manage deployment of services to each of the servers. 
 
-A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. The {KIE_SERVER} includes an embedded in-memory H2 database engine. This type of environment uses the least possible amount of resources. However, because of the in-memory database, restarting the {KIE_SERVER} pod leads to loss of all process information.
+If necessary, you can create separate environments, so that you can use one deployment of {CENTRAL} to author services (_authoring environment_) and another deployment of {CENTRAL} to manage deployment of staging or production services on several {KIE_SERVERS} (_managed server environment_). Usually, one {KIE_SERVER} is sufficient for a dedicated authoring environment. You can use an external Maven repository to store services from an authoring environment and deploy them to a separate managed server environment.
 
-An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. The database is provided by a separate pod. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are involved in authoring at the same time.
+For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. 
 
-You can also deploy additional managed or immutable {KIE_SERVERS}, if required. {CENTRAL} can automatically discover any {KIE_SERVERS} in the same namespace, including immutable {KIE_SERVERS} and managed {KIE_SERVERS}. This feature requires the `OpenShiftStartupStrategy` setting, which is enabled for all {KIE_SERVERS} except those deployed in a fixed managed infrastructure. For instructions about deploying managed {KIE_SERVERS} with the `OpenShiftStartupStrategy` setting enabled, see {URL_DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}[_{DEPLOYING_MANAGED_FREEFORM_ON_OPENSHIFT}_]. For instructions about deploying immutable {KIE_SERVERS}, see {URL_DEPLOYING_IMMUTABLE_ON_OPENSHIFT}[_{DEPLOYING_IMMUTABLE_ON_OPENSHIFT}_].
+Then, if necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create an environment with multiple {KIE_SERVERS}. This environment can be a managed server environment for staging and production deployment of services. 
```

Comment:
Just noting the "Then, ..." that IBM SG says to avoid. Can just exclude if you want, "If necessary, you can deploy ..." I don't care personally, just reminding of IBM SG aversion to starting sentences with "Then" and with the form, "do x, then do y." They say that it should always be "and then". Anyway, trivial.